### ISSUE TITLE
Brave Ad notifications are not shown on Windows until notifications are authorized

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#8843]
## Description 

Brave Ad notifications are not shown on Windows until notifications are authorized

## Steps to Reproduce

   1. View an ad notification

## Actual result:

Ad notification is not shown on Windows until notifications are authorized, i.e. visiting http://bennish.net/web-notifications.html and clicking "Allow", i.e.

<img width="360" alt="Screenshot 2020-03-24 at 19 21 16" src="https://user-images.githubusercontent.com/1188719/77468767-f86ce780-6e05-11ea-8d78-59e8a2f055dd.png">

I could not reproduce the issue after uninstalling Brave Browser and installing version 1.5.115 (Chromium: 80.0.3987.149 (Official Build) (64-bit). The only difference I could see was that Brave appeared in "Notifications & actions" within Windows settings whereas before I uninstalled the older build this option was missing

## Expected result:
Ad notification should be shown on Windows

## Reproduces how often: 

Intermittent issue

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel?
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N/A
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome? N/A

## Miscellaneous Information:

```
[9836:10792:0324/102454.662:INFO:notification_helper_win.cc(208)] Notifications are disabled
```
appeared in the debug logs caused by `src/chrome/browser/notifications/notification_platform_bridge_win.cc` failing with `Unable to initialize toast notifier`

**Comment by tmancey** [comment]
Fixed by #8932

**Comment by bsclifton** [comment]
Closing - looks like we accidentally forgot to close
cc: @tmancey
